core/test-src/org/pentaho/di/core/row/value/ValueMetaBaseTest.java

```diff
@@ -517,4 +517,17 @@ Date utc( int year, int month, int dat, int hrs, int min, int sec, int ms ) {
     cal.set( Calendar.MILLISECOND, ms );
     return cal.getTime();
   }
+
+  @Test
+  public void testGetNativeDataTypeClass() {
+    ValueMetaInterface base = new ValueMetaBase();
+    Class<?> clazz = null;
+    try {
+      clazz = base.getNativeDataTypeClass();
+      fail();
```

Comment:
@matthewtckr wingman is failing on this, maybe the imports have changed since your original commit?